### ISSUE TITLE
Gerald's bed is wandering quite far out of level. Query: Belt driving jacking screws slipping resulting in uneven adjustment of levels?

### ISSUE THREAD

**Issue by epsaul** [DoESLiverpool/somebody-should#250]

**Comment by amcewen** [comment]
As an interim measure, John H has shimmed the bed.  From his email:

"Alrite chaps, just a heads up that after a chat with Patrick I have made a temporary fix to Gerald.

Whilst the bed is still not level I've taken it to a mm or two by slotting stuff under his steel grate making the material a consistent distance from the laser

I ran test on 9mm and the laser cuts cleanly or a least gets close to partially cuts all the way through in all the areas I test in the bed.

I've taped a sign to Gerald asking people not to lift the grate so the wooden pieces supporting do not fall :) "

**Comment by DefProc** [comment]
I finally got round to fixing this — which is testament to how good a job John H did with the shims!

Although there was no clear reason why this happened (though the bed drive belts aren't the newest things), but my best guess is that the belts slipped off a pulley in use, and then re-seated as the pulleys turned (which they will tend to do). 

Along with a good vacuuming out and re-oiling the rails, I pried the transverse belt over one of the pulleys while running the motor, manually moved one side to be level with the other, and then ran the motor further (with my fingers well out of the way) to get the drive belt to seat back on both pulleys. 

The bed looks to be about 1mm difference across any two points.
